### ISSUE TITLE
Cover card not showing slider

### ISSUE THREAD

**Issue by zgentry** [UI-Lovelace-Minimalist/UI#795]
**Describe the bug**
After adding the cover card to 2 fresh installations of HA, the card is not showing anything in the slider section of the card. The area where the slider should be is there, but when the cover is open some percentage, the slider does not show. Changing from light mode to dark mode did not resolve the issue and there are no themes installed. This is with a fresh installation of UI Lovelace Minimalist with just the card added to the ui-lovelace.yaml.

**To Reproduce**

1. Install UI Lovelace Minimalist
2. Open the ui-lovelace.yaml 
3. Scroll to the bottom and add the following card

``` yaml
- type: "custom:button-card"
  template: "card_cover"  
  entity: cover.zebs_desk  
  variables:  
    ulm_card_cover_enable_controls: true  
    ulm_card_cover_enable_slider: true
```

5. Save the changes.
6. Open the UI Lovelace Minimalist dashboard.

**Expected behavior**
The bottom card would show a colored slider based on the open percentage.

**Screenshots**
![image](https://user-images.githubusercontent.com/83561050/175817473-12965547-1ef3-4066-b5ad-501edcf1c9e0.png)
![image](https://user-images.githubusercontent.com/83561050/175817484-f3d06d9c-d75e-4afe-a0fe-2541ab1ea6d0.png)

**Additional context**
When clicking in the location of the slider, it will still open/close the cover to the correct percentage, showing the slider is working. The only issue is the slider not showing on the cover card. The slider on a light card is working correctly.

I have also cleared cache on desktop, and the android app is showing the same issue.

**Comment by schumijo** [comment]
Hi @zgentry 

Can you confirm you have also installed "My Cards Bundle" card from HACS ?

**Comment by zgentry** [comment]
Hi @schumijo 

When installing UI Lovelace Minimalist I just followed the instruction set at https://ui-lovelace-minimalist.github.io/UI/setup/download/#install-repository for the download and installation using HACS. I also did use select the Include Custom Cards option during the installation. As such, I didn't install any "My Cards Bundle" from HACS and I do not see anything that was added automatically during the HACS download/installation.
